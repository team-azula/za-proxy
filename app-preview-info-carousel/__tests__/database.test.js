const mongoose = require('mongoose');
const Carousel = require('../database/model.js');
const data = require('../database/seed_data.js');
const seedData = require('../database/seed_data');

describe('database tests', () => {

  it('it will check the matchers and pass', () => {

    expect(seedData[0]).toMatchSnapshot({
      id: expect.any(Number),
      app_description: expect.any(String),
      additional_text: expect.any(String),
      images: expect.any(Array)
    });
  });


});






// describe('database tests', () => {

//   test('the seed script adds to the database', done => {
//     // The seed file has 3 Schemas for id 1
//     // Needs to be changed if more data is added
//     Schema.find({ id: 2 }, (err, ids) => {
//       expect(id.length).toBeGreaterThanOrEqual(0);
//       done();
//     });
//   });

//   test('will check the snapshot and pass with correct types', () => {

//     expect(seedData[9]).toMatchSnapshot({
//       id: expect.any(Number),
//       app_description: expect.any(String),
//       images: expect.any(Array)
//     });
//   });


//   it('will be able to find a specific id with .find()', () => {
//     seedData.find({id: 99}, (err, id) => {
//       expect(seedData[9].id).toEqual(99);
//       done();
//     })
//   })


//   test('the schema should have the correct ids', done => {
//     Schema.findOne({ id: 1 }, (err, id => {
//       expect(id).toEqual(
//         expect.objectContaining({
//           id: expect.any(String),
//           app_description: expect.any(String),
//           images: expect.any(Array)
//         })
//       );
//       mongoose.connection.close();
//       done();
//     });
//   });



// });
