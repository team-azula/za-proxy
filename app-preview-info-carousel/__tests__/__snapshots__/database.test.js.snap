// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`database tests it will check the matchers and pass 1`] = `
Object {
  "additional_text": Any<String>,
  "app_description": Any<String>,
  "id": Any<Number>,
  "images": Any<Array>,
}
`;
